packages/nextjs/components/impact-vector/ImpactvectorLists.tsx

```diff
@@ -0,0 +1,29 @@
+import ImpactVectorListCard from "./ImpactVectorListCard";
+import { useQuery } from "@tanstack/react-query";
+import { VectorList } from "~~/app/types/data";
+
+const ImpactvectorLists = () => {
+  const { data: vectorLists } = useQuery<VectorList[]>({
```

Comment:
I know you mentioned issues with SWR when trying to use it here but I wish we could give it another try and remove react-query entirely.

Reply:
ok i will do just that,